### ISSUE TITLE
Could not set unknown property 'jenkinsVersion' for extension 'jenkinsPlugin' of type org.jenkinsci.gradle.plugins.jpi.JpiExtension

### ISSUE THREAD

**Issue by wheelerlaw** [jenkinsci/gradle-jpi-plugin#171]
Here is my build.gradle:

```
plugins {
    id 'org.jenkins-ci.jpi' version '0.39.0'
}

group = 'org.cvp.pipeline.plugins'
version = '0.0.1'
description = 'A dummy plugin that just bundles dependencies that are made available on the Jenkinsfile classpath'

jenkinsPlugin {
    jenkinsVersion = '2.204.2'
}
```

When I try to refresh the project in IntelliJ, and when I try to build the project from the command line, I get this:

```
Build file '/home/wlaw/pipeline/plugins/dependencies/build.gradle' line: 11

A problem occurred evaluating project ':plugins:dependencies'.
> Could not set unknown property 'jenkinsVersion' for extension 'jenkinsPlugin' of type org.jenkinsci.gradle.plugins.jpi.JpiExtension.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

Running Gradle 6,7

**Comment by sghill** [comment]
Can you try version 0.40.0-rc.2 of the plugin?

This property isn’t available in 0.39.0 (coreVersion is).

**Comment by wheelerlaw** [comment]
> This property isn’t available in 0.39.0 (coreVersion is).

The documentation should be updated to reflect that:

```plugins {
  id 'org.jenkins-ci.jpi' version '0.39.0'
}

group = 'org.jenkins-ci.plugins'
version = '1.2.0-SNAPSHOT'
description = 'A description of your plugin'

jenkinsPlugin {
    // version of Jenkins core this plugin depends on, must be 1.420 or later
    jenkinsVersion = '1.420'
```

Looks like the property works with 0.40.0-rc.2.

**Comment by sghill** [comment]
Thanks for the report @wheelerlaw! I've released 0.40.0 and updated the README to reflect that version.
